tests/tce/multi_AS_copy/host.cpp

```diff
@@ -44,7 +44,7 @@
 #include "poclu.h"
 
 static char
-kernelSourceCode[] =
+kernelSourceCodeTCE[] =
```

Comment:
Please try if printf() works OK with TCE too. LLVM should optimize it at least to putstring which is OK too.